tests/test_events.py

```diff
@@ -0,0 +1,42 @@
+from lomond import events
+import pytest
+import six
+
+
+@pytest.mark.parametrize("event_object, expected", [
+    (events.Event(), 'Event()'),
+    (
+        events.Binary(b'\xef' * 17),
+        "Binary(%r + 1 bytes)" % six.b('\xef' * 16)
+    ),
+    (events.Binary(b'\x01'), "Binary(%r)" % six.b('\x01')),
```

Comment:
Ah, I get it. I think as a general rule it might be better to have an `if PY2` branch in tests where it differs. It just makes it black and white, even if it is an extra line or two.